deploy/kubernetes/csi-cvmfsplugin.yaml

```diff
@@ -11,31 +12,30 @@ spec:
       labels:
         app: csi-cvmfsplugin
     spec:
-      serviceAccount: csi-nodeplugin
+      serviceAccount: cvmfs-nodeplugin
       hostNetwork: true
       containers:
-        - name: driver-registrar
-          image: quay.io/k8scsi/driver-registrar:v0.3.0
+        - name: csi-driver-registrar
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.2
           args:
-            - "--v=5"
-            - "--csi-address=$(ADDRESS)"
-          env:
-            - name: ADDRESS
-              value: /var/lib/kubelet/plugins/csi-cvmfsplugin/csi.sock
-            - name: KUBE_NODE_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: spec.nodeName
+            - "--csi-address=/csi/csi.sock"
+            - "--kubelet-registration-path=/var/lib/kubelet/plugins/csi-cvmfsplugin/csi.sock"
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "rm -rf /registration/csi-cvmfsplugin /registration/csi-cvmfsplugin-reg.sock"]
           volumeMounts:
-            - name: socket-dir
-              mountPath: /var/lib/kubelet/plugins/csi-cvmfsplugin
+            - name: plugin-dir
+              mountPath: /csi
+            - name: registration-dir
+              mountPath: /registration
         - name: csi-cvmfsplugin
           securityContext:
             privileged: true
             capabilities:
               add: ["SYS_ADMIN"]
             allowPrivilegeEscalation: true
-          image: csi-cvmfsplugin:v0.3.0
+          image: gitlab-registry.nautilus.optiputer.net/prp/cvmfs-csi:latest
```

Comment:
Maybe we keep the base image name here? And latest in the default is maybe not ideal, keeping a tag there are no surprises for default deployments.

Reply:
Yes, the existing image is just what we used internally. Should not be the one referenced from the upstream.